Traveler/ViewControllers/AuthViewController.swift

```diff
@@ -52,6 +52,7 @@ extension AuthViewController: MainViewControllerDelegate {
         GIDSignIn.sharedInstance()?.signOut()
         Profile.clearStoredProfile()
 
+        Traveler.identify(nil, attributes: [:])
```

Comment:
The internal mechanism in the facade allows for attributes to be nil. The facade method does not, modify the signature so that the call can read Traveler.identify(nil, attributes: nil). Then it is clear that attributes are not required.

Reply:
Or better have a default value of `nil` for attributes so it renders that argument optional